build/release-tars/BUILD.bazel

```diff
@@ -168,6 +168,23 @@ pkg_tar(
     }),
 ) for arch in archs] for os, archs in CLIENT_PLATFORMS.items()]
 
+# This tarball contains the actual kubectl-cert-manager.tar.gz
+# This is meant to transport that file inside the build system
+# later to be extended with more requirements to use Krew.
+[[pkg_tar(
+    name = "krew-kubectl-cert-manager-%s-%s" % (os, arch),
+    srcs = [
+        ":kubectl-cert-manager-%s-%s" % (os, arch),
+    ],
+    extension = "tar.gz",
+    mode = "0644",
+    package_dir = "krew-kubectl-cert-manager-%s-%s" % (os, arch),
```

Comment:
I'd expect this to be where `client` as a subdirectory is introduced, so something like: `cert-manager-kubectl-cert_manager-%s-%s/client`?